test/test_parmed_residue.py

```diff
@@ -128,5 +130,20 @@ def testTermini(self):
         self.assertTrue(residue.DNAResidue.has('DG3'))
         self.assertFalse(residue.RNAResidue.has('G4'))
 
+    def testModifiedDNARNA(self):
+        """ Tests that modified DNA/RNA residue names are properly recognized"""
+        def count_nures(parm):
+            nnuc = 0
+            for res in parm.residues:
+                if RNAResidue.has(res.name) or DNAResidue.has(res.name):
+                    nnuc += 1
+            return nnuc
+
+        pdb_with_nnures = [('1EHZ', 76), ]
+        for pdbid, correct_nres in pdb_with_nnures:
+            pdb = pmd.download_PDB(pdbid)
+            nres = count_nures(pdb)
+            self.assertTrue(correct_nres == nres)
```

Comment:
Change this to:

```Python
self.assertEqual(correct_nres, nres)
```

It improves the failure message if it fails.  So instead of getting `AssertionError: False is not True`, you get `AssertionError: repr(correct_nres) is not equal to repr(nres)`.